controllers/noobaa-rules.yaml

```diff
@@ -1,104 +1,109 @@
      rules:
        - expr: |
-           NooBaa_num_buckets_claims{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_buckets_claims{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_buckets_claims
+         record: namespace:noobaa_buckets_claims:max
        - expr: |
-           NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_bucket_claims
+         record: namespace:noobaa_unhealthy_bucket_claims:max
        - expr: |
-           NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: job:noobaa_total_unhealthy_namespace_buckets
+         record: namespace:noobaa_unhealthy_namespace_buckets:max
        - expr: |
-           NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_accounts_num{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: job:noobaa_namespace_bucket_count
+         record: namespace:noobaa_accounts:max
        - expr: |
-           NooBaa_accounts_num{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_total_usage{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_accounts_num
+         record: namespace:noobaa_usage:max
        - expr: |
-           NooBaa_total_usage{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_total_usage
+         record: namespace:noobaa_unhealthy_namespace_resources:max
        - expr: |
-            NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_namespace_resources{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_namespace_resources
+         record: namespace:noobaa_namespace_resources:max
        - expr: |
-           NooBaa_num_namespace_resources{namespace="redhat-data-federation"}
+           count without (instance) (NooBaa_namespace_resource_status{namespace="redhat-data-federation"} == 1)
          labels:
            origin: data-federation-service
-         record: noobaa_num_namespace_resources
+         record: namespace:healthy_noobaa_namespace_resources:count
```

Comment:
I would expect that

```
namespace:healthy_noobaa_namespace_resources:count = namespace:noobaa_namespace_resources:max - namespace:noobaa_unhealthy_namespace_resources:max
```

so I'm not sure you need namespace:healthy_noobaa_namespace_resources:count  and namespace:noobaa_namespace_resources:max.

Reply:
We actually require them for the dashboard.